### ISSUE TITLE
EIP Tags/Labels/Topics

### ISSUE THREAD

**Issue by Pandapip1** [ethereum/EIPs#6037]
### Proposed Change

Add a new preamble item that allows EIPs to assign themselves "tags" so that users can filter by topic. For example:

```diff 
---
eip: 5484
title: Consensual Soulbound Tokens
description: Interface for special NFTs with immutable ownership and pre-determined immutable burn authorization
author: Buzz Cai (@buzzcai)
discussions-to: https://ethereum-magicians.org/t/eip-5484-consensual-soulbound-tokens/10424
status: Final
type: Standards Track
category: ERC
created: 2022-08-17
requires: 165, 721
+ tags: nft, soulbound
---

...
```

**Comment by Pandapip1** [comment]
> Would we have a defined list of tags?

No.

> How would we render this in jekyll?

With difficulty. It's certainly doable, though, with client-side rendering.

**Comment by SamWilsn** [comment]
> > Would we have a defined list of tags?
> 
> No.

I think I would be weakly opposed to this change then. I fear it would become just the description, but shorter. I do think the barrier to creating a tag should be quite low, however. That way it would be useful for the EOF or the SIWE family of EIPs.



> > How would we render this in jekyll?
> 
> With difficulty. It's certainly doable, though, with client-side rendering.

:rofl: I didn't really mean on a technical level. Like would we subdivide `Core / Final` by tag, for example?

**Comment by Pandapip1** [comment]
> I think I would be weakly opposed to this change then. I fear it would become just the description, but shorter. I do think the barrier to creating a tag should be quite low, however. That way it would be useful for the EOF or the SIWE family of EIPs.

I guess a small amount of tag curation could be useful. I'd be in support of that compromise.

> 🤣 I didn't really mean on a technical level. Like would we subdivide `Core / Final` by tag, for example?

Each tag would have a page (`/tag/<tag>`) that lists the relevant EIPs using the same template as the category-specific pages.

**Comment by Pandapip1** [comment]
Dismissing stale bot.

**Comment by Pandapip1** [comment]
Is the consensus that this is a good idea?

**Comment by Pandapip1** [comment]
I think that the consensus is that this is a good idea. I'll put together a proposal.

**Comment by Pandapip1** [comment]
Dismissing shale bot.

**Comment by Pandapip1** [comment]
Oops somehow didn't get the stale notification for this. Definitely want to keep this open!

**Comment by Pandapip1** [comment]
Superseded by the EIP Charter thing
